presentations/en/Micro-acquisition_PPOC.md

```diff
@@ -0,0 +1,274 @@
+
+- Through a **simplified and expedited** process that **reduces the barriers** that these organizations (including start-ups) and freelancers face, and a **targeted communications strategy** (to diverse businesses such as those that are Black, and Indigenous led), this pilot is **expected to increase access to IT talent that the GC is currently missing out on**
+- Why do we want to do this?  Because strategic procurement of open source code can bring knowledge into IT teams that is not currently there. Suppliers can be contracted to solve problems and when they turn over the open source code, ESDC IT teams will be able to learn from suppliers in almost a train the trainer sort of way. This would enable IITB to build capacity in-house and embed modern tools and practices within IITB teams.
+- Further, contributions to GC open source projects, or open source projects being leveraged by the GC, has the **potential to be leveraged as a recruitment tool** for IITB. (CDS has experienced this with COVID Alert)
+
+---
+
+## Alignment
+
+- ESDC Mandate
+- Digital Operations Strategic Plan
+- Digital Nations Charter commitment
+- Directive on Service and Digital
+- ESDC Investment Plan
+
+Note:
+
+- there is strong alignment to policy and mandate for this experimental pilot ESDC's mandate of "works to improve the standard of living and quality of life for all Canadians. We do this by promoting a labour force that is highly skilled. We also promote an efficient and inclusive labour market." It makes ESDC a good fit to lead this initiative. Especially during COVID when many are facing unemployment and underemployment.
+- Digital Operations Strategic Plan – which speaks to the need for Procurement Modernization for IT specifically, Agility, and Open Source​
+- Digital Nations Charter commitment (formerly D9) – Open Standards, Open Source, Open Markets, Open Government, and which Canada is a signatory
+- Directive on Service and Digital – Open First​
+- ESDC Investment plan (2017-18 to 2022-23) – Emphasizes the use of OSS​
+
+---
+
+## Expected Results
+
+- Increased participation and access to developers who wouldn’t normally bid on GC IT contracts
+- Increased working in the open by ESDC
+- Reduced workload burden on ESDC procurement officers
+- Increased ESDC capabilities around agile, including breaking work up into smaller chunks
+- Increased speed of payment of suppliers
+
+Note:
+
+- reduced burden on procurement officers means they can focus on high risk, complex procurements
+- breaking work out into smaller work packages enables same-day delivery and drives high organizational performance
+
+---
+
+## Forecasted costs
+
+<table>
+ <tr>
+   <th>Fiscal Year</th>
+   <th>IITB Salary*</th>
+   <th>CFOB Salary*</th>
+   <th>Total Salary</th>
+   <th>Non-Salary</th>
+   <th>Total</th>
+ </tr>
+ <tr>
+   <td>2020-21</td>
+   <td>$81K</td>
+   <td>$15K</td>
+   <td>$96K</td>
+   <td>$0.015</td>
```

Comment:
We need to disclose a cost of 1,5 cents?

Reply:
Another good catch! There is a 'K' missing from that line.  I've made that fix. So its $15 and not 1.5 cents. Still not really worthy of disclosing but it is in the costing worksheet and so figured the two things would need to match. :-)